src/flows/pipes/Notification/view.tsx

```diff
@@ -172,7 +174,7 @@ const Notification: FC<PipeProp> = ({Context}) => {
             p.lineNumber,
             p.column
           ),
-          text: ` r.${exp} `,
+          text: ` \$\{r.${exp}\} `,
```

Comment:
just wondering, why this change?

Reply:
I originally forgot that the variables are supposed to be put into the alert message like this: `${r.column}` instead of `r.column`